Tailer-Platform/tailer-studio
# Improve execution date & time selection

## Related issue(s)

Closes #345 


## Description

This PR improves execution date & time selection.


## What does this PR do?

- [x] Set current date as default value for date
- [x] Add default value on time (`00:00`)


## Deploy notes

Review & merge.


## Steps to test or reproduce

1. Go to a `tables-to-tables` configuration
2. Click on `LAUNCH`
3. Enter date and time
4. Dag should be launched correctly
